azurelinuxagent/common/exception.py

```diff
@@ -66,8 +66,9 @@ class ExtensionDownloadError(ExtensionError):
     When failed to download and setup an extension
     """
 
-    def __init__(self, msg=None, inner=None):
+    def __init__(self, msg=None, inner=None, code=-1):
```

Comment:
This change is a part of the code cleanup - the parameter code was missing (this Exception extends from `ExtensionError` and should have code param in its `__init__()`

Reply:
Shouldn't you instead pass code to super as the third parameter?

Reply:
Fixed. Thanks.